sw/device/silicon_creator/lib/drivers/spi_device.h

```diff
@@ -37,7 +37,7 @@ enum {
   /**
    * LSB of the 2-byte device ID.
    */
-  kSpiDeviceJedecDensity = 0x0a,
+  kSpiDeviceJedecDensity = 20,
```

Comment:
nit: add a comment that the density is expressed as log2(size of flash in bytes).

Reply:
Done, thanks again for catching this! Also added and `static_assert` in `spi_device.c`.

Reply:
Update: english_breakfast build failed due to smaller flash. I moved this check to `spi_device_unittest` in order not to have a `#ifndef OT_IS_ENGLISH_BREAKFAST` in `silicon_creator` code.